### ISSUE TITLE
Upon creating submission user is sometimes directed to the wrong submission

### ISSUE THREAD

**Issue by jdhayhurst** [EBISPOT/gwas-deposition-ui#331]
Twice in sandbox, I have clicked "create submission" for a published "open for summary statistics submission" and rather than being directed to the correct submission, I have been directed to this one instead: 6062e31837158e00011d672e.

**Comment by ljwh2** [comment]
Is this the same issue as goci#554?

**Comment by ljwh2** [comment]
@jdhayhurst has this happened since?

**Comment by jdhayhurst** [comment]
@ljwh2 I don't think this is the same as goci#554 - @earlEBI can probably confirm this.
I haven't witnessed it happening again but it doesn't happen all of the time.

**Comment by earlEBI** [comment]
@ljwh2  Yes, same issue. I'm pretty sure it's happened every time I've created a summary statistics-only submission.
